cohesity_management_sdk/samples/add_vm_to_protection_and_run/add_vm_to_protection_and_run.py

```diff
@@ -51,15 +51,15 @@ def check_vms_exists(self, vm_list):
         """
         str_vm_list = ','.join(vm_list)
         # Note: This API works only with vCenter VMs.
-        print str_vm_list
+        print(str_vm_list)
```

Comment:
Are we supporting python3 only, this change indicates python2 to python3 conversion